benchmark/realnest/14_list_sort_flatten_slice.benchmark

```diff
@@ -0,0 +1,21 @@
+# name: benchmark/realnest/14_list_sort_flatten_slice.benchmark
+# description: List sort flattened lists
+# group: [realnest]
+
+name list_sort_flatten_slice
+group real_nest
+
+require json
+
+require httpfs
+
+cache real_nest.duckdb
+
+load benchmark/realnest/load.sql
+
+run
+SELECT list_sort(flatten(list_slice(list(Jet), 150, 200, 3)), 'ASC'),
```

Comment:
Same here, I don't quite understand what is getting benchmarked. 

`list(Jet), list(Muon), list(Photon), list(Tau)` all create on big list from the row lists. I think it's better if we make list_slice operate on a row level basis. You can create ta copy of the run2012B_singleMu table by placing the following query in the load.sql

```
create or replace table single_mu_lists as select * REPLACE(list_resize(Jet, 10, NULL) as Jet, list_resize(Muon, 10, NULL) as Muon, list_resize(Photon, 10, NULL) as Photon, list_resize(Tau, 10, NULL) as Tau) from run2012B_singleMu;
```

The benchmark query could then look like this
```
SELECT list_sort(list_slice(Jet, 2, 9), 'ASC'),
   list_sort(list_slice(Muon, 1, 6), 'DESC'),
    list_sort(list_slice(Photon, 1, 3), 'ASC'),
    list_sort(list_slice(Tau, 5, 9), 'DESC')
FROM run2012B_singleMu;
```

and you'd get the same number of rows as in the input table. @taniabogatsch would this make more sense? Or should the benchmark just stay as is?

Reply:
I think a dedicated benchmark for slicing is excellent. Also, we can omit the `list_sort`, since we are already benchmarking it in a different file (IIRC).

Reply:
Resolved